src/Detector/DotNetCore/DotnetCoreConstants.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Microsoft.Oryx.Detector.DotNetCore
 {
-    public static class DotNetCoreConstants
+    internal static class DotNetCoreConstants
```

Comment:
Throughout this PR I have made types `internal` which I felt are not required to be exposed to clients as they are internal details.